src/Frontend/Core.hs

```diff
@@ -420,18 +420,25 @@ instance Page CsrfToken where
     -- token. However since this is just used for testing purposes so far there is no need to allow
     -- that.
 
-data Page404 = Page404
+data ServerErrorPage = Page404 | Page5xx
```

Comment:
`HttpErrorPage`?  Could be renamed together with `catchErrors` for consistency, but i don't insist.